.gitignore

```diff
@@ -0,0 +1,54 @@
+*.gem
+*.rbc
+/.config
+/coverage/
+/InstalledFiles
+/pkg/
+/spec/reports/
+/spec/examples.txt
+/test/tmp/
+/test/version_tmp/
+/tmp/
+
+# Used by dotenv library to load environment variables.
+# .env
+
+## Specific to RubyMotion:
+.dat*
+.repl_history
+build/
+*.bridgesupport
+build-iPhoneOS/
```

Comment:
Taken from here 
https://github.com/github/gitignore/blob/master/Ruby.gitignore
And forgot to add a comment line.

spec, test and temp i think is useful in the future.
We can remove instead iPhone and other stuff, but isn't better to have a reference?